bin/set-platformsh-vars.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+PSH_PROJECT_ID=$1
+PSH_ENVIRONMENT=${2-master}
+
+if [ -z "$PSH_PROJECT_ID" ]; then
+    echo ""
+    echo "ERROR: Please provide Platform.sh Project Id"
+    echo ""
+    echo "Available projects:"
+    platform project:list --columns id,title 2>/dev/null
+    exit 1
+fi
+
+export PLATFORM_ROUTES=$(platform ssh -p ${PSH_PROJECT_ID} -e ${PSH_ENVIRONMENT} -A app -q 'echo $PLATFORM_ROUTES')
+platform tunnel:open -q -p ${PSH_PROJECT_ID} -e ${PSH_ENVIRONMENT} -A app
```

Comment:
The `-q` means this will silently hang if you're opening a tunnel to `master`, because the CLI asks for confirmation (without `-y`).

Silently opening a tunnel to the `master` environment seems risky: you might imagine you're writing to a DB on your local (or that you're writing to a dev env), when you're actually writing to production.